python_src/tests/performance_manager/test_performance_manager.py

```diff
@@ -91,6 +93,7 @@ def fixture_db_manager() -> DatabaseManager:
     """
     set_env_vars()
     db_manager = DatabaseManager()
+    run_alembic_migration("performance_manager")
```

Comment:
Create schema with alembic for testing.

Reply:
i don't think we need to run the data migration here since we're spinning it up against a blank db everytime.

Reply:
If we don't run the migration there won't be able tables in the blank db to load data into. This is how schema is created now.